org.adempiere.base/src/org/compiere/process/SvrProcess.java

```diff
@@ -731,7 +732,7 @@ private void unlock ()
 			MPInstance mpi = new MPInstance (getCtx(), m_pi.getAD_PInstance_ID(), null);
 			if (mpi.get_ID() == 0)
 			{
-				log.log(Level.SEVERE, "Did not find PInstance " + m_pi.getAD_PInstance_ID());
+				log.log(Level.INFO, "Did not find PInstance " + m_pi.getAD_PInstance_ID());
 				return;
 			}
 			mpi.setIsProcessing(false);

```

Comment:
this whole block should take a similar approach as above, i.e check that m_pi.getAD_PInstance_ID() > 0.